data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1924 @@
+				`	"Alright, which worlds do you need me to land on?"`
+				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
+					decline
+			`	They ask you to show them your map of the galaxy, so you bring them inside your ship and point out to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`
+			`	You give them a quick rundown of each Quarg world, and after discussing among themselves for a bit, the Saryd speaks. "Interested in the Quarg's mining operations, and the various construction stages of their ringworlds, the consul is."`
+			`	He points to some of the worlds, and you list the places you need to land: <stopovers>.`
+			`	They call for some engineers to come work on your ship's sensors, the whole thing taking just a few minutes. When they're done, the Heliarch agents thank you for your cooperation, and tell you to head for the <destination> when you've collected all the data.`
+				accept
+	to fail
+		"reputation: Quarg" < 0
+	on complete
+		event "heliarch recon break" 10 21
+		"coalition jobs" ++
+		payment 165000
+		conversation
+			`The Heliarch trio are eagerly waiting for you when you return. They are quick to welcome you, urging you to show them your ship's logs so they can see what information you've gathered. They don't seem to pay much mind to data on Grakhord or Forpelog, being much more interested in the ringworlds. As they examine the data, though, the Saryd frowns. The Kimek brings all his legs closer to his body, and the Arach squints all her eyes - probably their versions of the Saryd's reaction.`
+			`	"Land on the rings, you really did, right Captain?" the Saryd asks while pointing at the places you visited.`
+			`	You nod, and the Kimek hits your control panel lightly with one of his legs. "No data on the rings, the sensors collected! Gone wrong, something must have."`
+			`	They have engineers return your sensors to normal, while you explain how your ship behaved while landing on them. They speak among themselves for a while, then the Arach says, "Too straightforward an approach, we might have taken. Captain <last>, contact you again soon, we will."`
+			`	They hand you <payment>, and leave to one of the Heliarch sections of the ring.`
+
+mission "Heliarch Recon Alta Hai"
+	invisible
+	landing
+	source "Alta Hai"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 1: active"
+	on offer
+		conversation
+			`Coming in for a landing, something from inside your ship produces an irritating, high pitched whistle sound. It grows stronger the closer you get to the ringworld, until your ship completely shuts down. Your controls are rendered useless as you attempt to land manually, only for your autopilot to click back to normal and land as usual at the last moment. Looking outside, you spot several Quarg standing still, staring at your ship, until they hastily leave without contacting you at all.`
```

Comment:
```suggestion
			`Coming in for a landing, something from inside your ship produces an irritating, high pitched whistle sound. It grows stronger the closer you get to the ringworld, until your ship completely shuts down. Your controls are rendered useless as you scramble to land manually, only for your autopilot to click back to normal and land as usual at the last moment. Looking outside, you spot several Quarg standing still, staring at your ship, until they hastily leave without contacting you at all.`
```
Personal preference. The shutdown seems sudden and `<first>` doesn't have to manually land often, so I feel this captures the situation better.